pkg/cmd/runsGet.go

```diff
@@ -99,6 +99,12 @@ func (cmd *RunsGetCommand) createCobraCommand(
 	runsGetCobraCmd.PersistentFlags().StringVar(&cmd.values.result, "result", "", "A filter on the test runs we want information about. Optional. Default is to display test runs with any result. Case insensitive. Value can be a single value or a comma-separated list. For example \"--result Failed,Ignored,EnvFail\"")
 	runsGetCobraCmd.PersistentFlags().BoolVar(&cmd.values.isActiveRuns, "active", false, "parameter to retrieve runs that have not finished yet.")
 
+	runsGetCobraCmd.MarkFlagsMutuallyExclusive("name", "age")
+	runsGetCobraCmd.MarkFlagsMutuallyExclusive("name", "requestor")
+	runsGetCobraCmd.MarkFlagsMutuallyExclusive("name", "result")
+	runsGetCobraCmd.MarkFlagsMutuallyExclusive("name", "active")
+	runsGetCobraCmd.MarkFlagsMutuallyExclusive("result", "active")
```

Comment:
Wondering what the error message looks like when you try this out ?

Reply:
replied in slack

Reply:
```
galasactl-darwin-arm64 runs get --name C18202 --requestor galasa                    
Error: if any flags in the group [name requestor] are set none of the others can be; [name requestor] were all set
```

Ok. That looks clear enough what's going wrong.

Do we get a non-zero exit code also when that happens ?

Reply:
yep, seems so if I've done that right
```
> galasactl-darwin-arm64 runs get --name C1802 --age 2d
Error: if any flags in the group [name age] are set none of the others can be; [age name] were all set
> echo $?
1
```